recipes/numdifftools/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "numdifftools" %}
+{% set version = "0.9.20" %}
+{% set sha256 = "443b43e43c970b4f37cd2a5a57f842192da89daa2d58105ca5f5f894b7d4fcd0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-py2.py3-none-any.whl 
+  url: https://files.pythonhosted.org/packages/py2.py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py2.py3-none-any.whl 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # I build from the wheel so that we don't need pyscaffold. The version
+  # needed by the source distribution of numdifftools is not compatible with conda-forge
+  script: pip install {{ name }}-{{ version }}-py2.py3-none-any.whl --no-deps
+  noarch: python
+  
+requirements:
+
+  build:
+  
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.9
+    - scipy >=0.8
+    - algopy >=0.4
+    
+  run:
+    - python
+    # This is needed because __init__ use pkg_resources
+    - setuptools
+    - numpy >=1.9
+    - scipy >=0.8
+    - algopy >=0.4
+
+test:
+
+  imports:
+    - numdifftools
+    - numdifftools.tests
+
+about:
+  home: "https://github.com/pbrod/numdifftools/"
+  license: BSD-3-Clause
+  summary: Solves automatic numerical differentiation problems in one or more variables.
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  doc_url: http://numdifftools.readthedocs.io/en/latest/
+  dev_url: https://github.com/pbrod/numdifftools/
+
+extra:
+  recipe-maintainers:
+    - giacomov
+    - pbrod
```

Comment:
@pbrod are you ok with being a maintainer?

Reply:
(he answered already in issue [#33](https://github.com/pbrod/numdifftools/issues/33) on the numdifftools repo)

Reply:
I am ok with this. ;-)